project/api/models.py

```diff
@@ -37,7 +39,8 @@ class TerritorialEntity(models.Model):
     color = ColorField()
     admin_level = models.PositiveIntegerField()
     inception_date = models.DecimalField(decimal_places=1, max_digits=10)
-    dissolution_date = models.DecimalField(decimal_places=1, max_digits=10)
+    dissolution_date = models.DecimalField(decimal_places=1, max_digits=10, blank=True, null=True)
+    stv_count = models.IntegerField(default=0, null=True, blank=True)
```

Comment:
I thought this would be done via aggregate function, like it was done for the votes in #102 
https://github.com/chronhq/backend/blob/7b75b9df04ef1da52b6a813d2ccc5089afdfa266/project/api/serializers.py#L200-L208